pkg/detectors/instamojo/instamojo.go

```diff
@@ -0,0 +1,105 @@
+package instamojo
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"net/http"
+	"regexp"
+	"strings"
+
+	"github.com/trufflesecurity/trufflehog/v3/pkg/common"
+	"github.com/trufflesecurity/trufflehog/v3/pkg/detectors"
+	"github.com/trufflesecurity/trufflehog/v3/pkg/pb/detectorspb"
+)
+
+type Scanner struct {
+	client *http.Client
+}
+
+// Ensure the Scanner satisfies the interface at compile time.
+var _ detectors.Detector = (*Scanner)(nil)
+
+var (
+	defaultClient = common.SaneHttpClient()
+	// Make sure that your group is surrounded in boundary characters such as below to reduce false positives.
+	//KeyPat is client_id
+	keyPat    = regexp.MustCompile(`\b([0-9a-zA-Z]{40})\b`)
```

Comment:
@fumblehool Just to be extra cautious, could we also surround this with `detectors.PrefixRegex` like you did with `secretPat`?

Reply:
@zricethezav
I've updated it to `keyPat    = regexp.MustCompile(detectors.PrefixRegex([]string{}) + `\b([0-9a-zA-Z]{40})\b`)`
because in the test case we pass `You can find a instamojo secret <secret> within id <id>`

Reply:
@fumblehool 

> keyPat    = regexp.MustCompile(detectors.PrefixRegex([]string{}) + \b([0-9a-zA-Z]{40})\b)

`instamojo` should be inside of the string array otherwise this isn't helping